shell/monitor/monitor.sh

```diff
@@ -2,11 +2,16 @@
 
 declare _monitor_pid _monitor_experiment_id _monitor_log_dir _monitor_update_es _monitor_current_test_id
 
-declare -r _monitor_eshost=aloja.bsc.es:9200
+declare -r _monitor_eshost=aloja.bsc.es
 declare -r _monitor_experiment_index=metadata
 declare -r _monitor_test_index=tests
 declare -r _monitor_mapping=aloja
 
+declare -r _monitor_proxy_email=bscaloja@gmail.com
+declare -r _monitor_proxy_password=foobar
```

Comment:
Hard-coding these credentials into the monitor seems like a bad idea, they should be passed in at runtime, e.g. using environment variables.